app/src/main/java/net/gsantner/markor/format/todotxt/TodoTxtTextActions.java

```diff
@@ -62,10 +59,13 @@ protected ActionCallback getActionCallback(@StringRes int keyId) {
     @Override
     public List<ActionItem> getActiveActionList() {
 
+
+        final int projectIcon = _appSettings.isTodoTxtAlternativeNaming() ? R.drawable.ic_local_offer_black_24dp : R.drawable.ic_baseline_add_24;
```

Comment:
Chose icon based on naming

Reply:
I would say one icon (the existing one) is fine, anyway the 99% will have the project&tag one.